tests/Makefile

```diff
@@ -2,7 +2,7 @@
 
 APIGEE_ACCESS_TOKEN ?= $(shell export SSO_LOGIN_URL=https://login.apigee.com && eval get_token -u $(APIGEE_USERNAME))
 
-PYTEST_CMD = APIGEE_ACCESS_TOKEN=$(APIGEE_ACCESS_TOKEN) INTERACTION_ID=$(INTERACTION_ID) poetry run pytest -s -v --color=yes --api-name=gp-connect-access-record-fhir --proxy-name=$(PROXY_NAME)
+PYTEST_CMD = APIGEE_ACCESS_TOKEN=$(APIGEE_ACCESS_TOKEN) INTERACTION_ID=$(INTERACTION_ID) poetry run pytest -s -v --color=yes --api-name=gp-connect-access-record-fhir --proxy-name=$(PROXY_NAME) --apigee-app-id=$(APIGEE_APP_ID) --apigee-organization=nhsd-prod
```

Comment:
Would it be possible to use this target locally? when `APIGEE_APP_ID` is not present?